packages/odyssey-storybook/src/guidelines/DocsStatus.stories.mdx

```diff
@@ -0,0 +1,233 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Guidelines/Docs Status" />
+
+# Docs Status
+
+This document serves as a scratch pad for the docs migration effort.
+
+"Current Status" provides a glimpse-friendly look at where the migration effort is at.
+
+The "General Items" section below applies to either all stories or Storybook generally.
+
+The "Components" section lists special considerations specific to each story. More often than not, this will entail missing or mismatched stories.
+
+Regarding stories: the "Existing" headings inventory those stories that exist in SB today, along with any caveats. "Assumed" inventories the stories assumed by the previous docs.
+
+## Current Status
+
+Docs content has been migrated as-is and is undergoing an audit for parity and quality.
+
+The result of this audit will be an inventory of items to discuss that can be turned into discrete, actionable tickets.
+
+## General Items
+
+### Missing
+
+- Component Anatomy is missing throughout.
+- Mismatch between SB typography and React typography.
+- All Component docs pages should surface an ArgsTable
+  - Are there any args, per component, that we want to hide?
+
+#### Components
+
+The following components are missing documentation:
+
+- Circular Load Indicator
+- Field
+- Field Group
+- Form
+- Heading
+- Icon
+- List
+- Screen Reader Text
+- Text Area (previously with Text Input
+
+#### Guidelines
+
+The following guidelines have not been migrated:
+
+- Color
+- Design Tokens
+- Elements
+- Typography
+- Iconography
+- Updates
+- Help
+- Figma link
+
+### Broken
+
+- Links need to be rewritten throughout.
+
+### Open Questions
+
+- How do we want to style internal reference tables? Using `<table>` for SB styling? Using `Table` for React dogfood?
+- How do we author stories for Bad Examples?
```

Comment:
what do you mean? wouldnt bad examples live within the docs for their component?

Reply:
That's my proposed plan. I've tried to stop myself from suggesting solutions during this first pass.

As we discussed in OH, I think authoring Docs-only stories in MDX is the solve here.

Reply:
@arnoldsandoval-okta I've added a proposals section to make this clear. Open Questions now contains only truly open questions.